types/minimatch-all/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "@types/minimatch": "*"
```

Comment:
I think u might of meant to add `minimatch` as a dependency instead of `@types/minimatch`. Similar to what the failed CI message says:
```
Error: In package.json: Dependency @types/minimatch not in the allowed dependencies list.
Don't use a 'package.json' for @types dependencies unless this package relies on
an old version of types that have since been moved to the source repo.
For example, if package *P* used to have types on Definitely Typed at @types/P,
but now has its own types, a dependent package *D* will need to use package.json
to refer to @types/P if it relies on old versions of P's types.
In this case, please make a pull request to microsoft/DefinitelyTyped-tools adding @types/P to `packages/definitions-parser/allowedPackageJsonDependencies.txt`.
```
but I'm not sure exactly. Like the CI says, u might need to edit `allowedPackageJsonDependencies.txt`

Reply:
no, this is correct, I want to have the old types not the ones from the new package